### PR TITLE
Change xml file

### PR DESCRIPTION
**Pull request by mnbp1371**

### CODE REVIEW

**Comment by mnbp1371** [comment]
@vispamir

File: rsfpidpay/rsfpidpay.xml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-	<name>System - RSForm! Idpay</name>
+	<name>System - IDPay for RSForm Pro</name>
 	<title>RSForm! Idpay</title>
-	<author>Trangell</author>
-	<creationDate>July 2020</creationDate>
-	<copyright>Copyright 2020 Trangell Team</copyright>
-	<license>http://www.gnu.org/licenses/old-licenses/gpl-2.0.html GNU/GPL version 2</license>
-	<authorEmail>info@trangell.com</authorEmail>
-	<authorUrl>http://trangell.com</authorUrl>
+	<author>mnbp1371</author>
```
Comment (vispamir) [inline rsfpidpay/rsfpidpay.xml]: Change author parameters idpay info. like as:

Developer: mnbp1371, vispa , Publisher: IDPay

File: rsfpidpay/rsfpidpay.xml [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+	<creationDate>August 2020</creationDate>
+	<copyright>Copyright 2020 IDPay</copyright>
+	<license>http://www.gnu.org/licenses/gpl-2.0.html GPLv2 or later</license>
+	<authorEmail>mohammadnabipour1371@gmail.com</authorEmail>
```
Comment (vispamir) [inline rsfpidpay/rsfpidpay.xml]: Change to support@idpay.ir

**Review by vispamir — CHANGES_REQUESTED** [review]

**Comment by mnbp1371** [comment]
Change author email

@vispamir

**Comment by mnbp1371** [comment]
Change author parameters
@vispamir

**Pull request merged**